README.md

```diff
@@ -188,6 +188,38 @@ For the deidentification of the EHR extracts, we rely on an instance running the
 [CogStack API](https://cogstack.org/) with a `/redact` endpoint. The URL of this instance should be
 set in `.env` as `COGSTACK_REDACT_URL`.
 
+### 3. Configure a new project
```

Comment:
did you want this to be in here?

Reply:
Oh ffs that's where that went 🤦‍♂️ 🤦‍♂️ 🤦‍♂️ 🤦‍♂️  I thought I'd lost that
> Note to self: `git stash --all` before switching branches